CONTRIBUTING.md

```diff
@@ -113,6 +113,11 @@ following rules before you submit a pull request:
    scale in dimensionality: n_features is expected to be lower than
    100".
 
+-  If your PR includes changes to files under the ``doc/`` or
+   ``examples/`` folders, include ``[doc build]`` in your commit message.
+   Otherwise, include ``[doc skip]`` in your commit message. This will 
```

Comment:
Thanks for making the quick range. I just have one more suggestion.
'message. This will'  -->  'message which will'.
Also,  you need to add the change in the same manner in `doc/developers/contributing.rst`